docs/developer/architecture/core/http-service.asciidoc

```diff
@@ -12,7 +12,7 @@ The service allows plugins to:
 * to execute custom logic on an incoming request or server response.
 * implement custom authentication and authorization strategy.
 
-See {kib-repo}blob/{branch}/docs/development/core/server/kibana-plugin-core-server.httpservicesetup.md[HTTP service API docs]
+Refer to {kib-repo}/blob/8.9/packages/core/http/core-http-server/src/http_contract.ts[HTTP service contract types].
```

Comment:
Points to concrete branch to mitigate broken links in the immediate term caused by code that's changed location